packages/ui/src/components/modal/theme/Modal.base.theme.ts

```diff
@@ -33,6 +33,7 @@ const theme: ModalTheme = {
       else if (props.size === 'sm') classes.push('sm:max-w-sm')
       else if (props.size === 'lg') classes.push('sm:max-w-xl')
       else if (props.size === 'xl') classes.push('sm:max-w-3xl')
+      else if (props.size === 'full') classes.push('sm:max-w-6xl')
       else classes.push('sm:max-w-lg')
```

Comment:
I would rather increase them all a little bit, and let full be full width, wdyt?
```suggestion
      else if (props.size === 'md') classes.push('sm:max-w-lg')
      else if (props.size === 'lg') classes.push('sm:max-w-3xl')
      else if (props.size === 'xl') classes.push('sm:max-w-6xl')
```